recipes-core/iotedge-daemon/iotedge-daemon_1.0.10.bb

```diff
@@ -244,9 +260,9 @@ SRCREV_FORMAT .= "_tokio-uds-windows"
 SRCREV_tokio-uds-windows = "${AUTOREV}"
 EXTRA_OECARGO_PATHS += "${WORKDIR}/tokio-uds-windows"
 
-LIC_FILES_CHKSUM=" \
+LIC_FILES_CHKSUM = " \
```

Comment:
This is a style change and should reside in a separated commit. Same for iotedge-cli. In the end you should have two commits, one that upgrades all iotedge recipes to 1.0.10, and another one that may fix styling issues.

Reply:
You previously asked me to remove the whitespaces in `recipes-core/iotedge-daemon/iotedge-daemon.inc`...
> You may remove this unnecessary whitespace.

Also...I had some difficulties squashing the changes, mainly due to the fact that I forked the zeus branch before my previous pr was approved. Would it be ok to disregard this pr entirely and I'll create a new one? If you want me to do that, would you like `recipes-core/iotedge-daemon/iotedge-daemon.inc` w/ or w/o line-breaks?